src/lib/ctable.lua

```diff
@@ -102,6 +102,16 @@ local function parse_params(params, required, optional)
    return ret
 end
 
+-- This gets 16 bits of salt, which should be enough to prevent accidental
```

Comment:
Since our hashes are 32 bits I think probably we should just get 32 bits of salt -- otherwise I think we don't actually solve the problem.  In difference to the rust hash table, we map [0, HASH_MAX) to the [0, size) domain by scaling, not modulo, so high bits are important too.